### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by barkasn**
### Purpose
Update test documentation

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -1,7 +1,29 @@
+This folder contains testing infrastructure for the pipelines in the skylab repository.
```
Comment (ekiernan) [inline test/README.md:1]: Could we add a little more description here for someone who should wonder in? What does testing infrastructure imply? Some questions we could answer in this overview that I might need your help:
1) What are the tests for/what do they actually do?
2) How do these tests differ from those that are used for validation? For example, how are the configuration files used here different than those we present on Terra?

File: test/README.md [hunk 2]
```diff
@@ -1,7 +1,29 @@
...
+
+**Description of Tests**
+- optimus: Tests related to the single-cell RNA-seq version of Optimus
+  - 4kpbmc: Runs the full 4kpbmc dataset with the sc variant of Optimus. Checksums are not provided (manual)
+  - pr: Run a small dataset derived from the 4k PBMC dataset with Optimus. It performs delta testing on the output matrix
```
Comment (ekiernan) [inline test/README.md:6]: Question- what does pr mean in this context?

File: test/README.md [hunk 3]
```diff
@@ -1,7 +1,29 @@
+This folder contains testing infrastructure for the pipelines in the skylab repository.
+
+**Description of Tests**
```
Comment (ekiernan) [inline test/README.md:3]: Add a brief description (edit the example below for accuracy):
"When a pull request (PR) is made for the Skylab Rep, the following tests are triggered automatically, except for tests marked as manual. " These tests will need to be run using the instructions in the 'Running tests manually' section."

File: test/README.md [hunk 4]
```diff
@@ -1,7 +1,29 @@
...
+  - pr: Runs a single-cell throught the SS2 pipeline in paired-end mode and checks the output files
+  - pr_single_end: Runs a single-cell throught the SS2 pipeline in single-end mode and checks the output files
+
 **Running Tests Manually**
 1) Set the environment variable BROAD_CROMWELL_KEY to have the contents of the cromwell credentials key JSON file.
 2) cd to the top level directory of the repository
-3) run the following command ```./test/trigger_test.sh [name of workflow without trailing /pr/]```
+3) run the following command ```./test/trigger_test.sh test_name [test_variant]```, where test_name is a directory name (e.g. ```optimus```) and test_variant is a nested directory name (e.g. ```4kpbmc```). test_variant is optional, if not provided the ```pr``` test will be run.
 
 Example:
```
Comment (ekiernan) [inline test/README.md:28]: Can we change the example to include the optional test_variant 4kpmc? 
 
Example with optional test_variant parameter:

```export BROAD_CROMWELL_KEY=`cat ~/identities/credentials.json`
cd ~/skylab/
./test/trigger_test.sh optimus 4kpbmc
```

File: test/README.md (hunk 3 above)
Comment (ekiernan) [reply test/README.md:3]: Change to:
## Description of tests and optional test variants

File: test/README.md (hunk 1 above)
Comment (barkasn) [reply test/README.md]: Updated

File: test/README.md (hunk 2 above)
Comment (barkasn) [reply test/README.md:6]: Yes, also included in intro

File: test/README.md (hunk 4 above)
Comment (barkasn) [reply test/README.md:28]: updated

**Pull request merged**